open-ce/conda_utils.py

```diff
@@ -0,0 +1,34 @@
+"""
+*****************************************************************
+Licensed Materials - Property of IBM
+(C) Copyright IBM Corp. 2020. All Rights Reserved.
+US Government Users Restricted Rights - Use, duplication or
+disclosure restricted by GSA ADP Schedule Contract with IBM Corp.
+*****************************************************************
+"""
+import os
+
+import conda_build.api
+from conda_build.config import get_or_merge_config
+import conda_build.metadata
+
+def render_yaml(path, variants=None, variant_config_files=None):
+    """
+    Call conda-build's render tool to get a list of dictionaries of the
+    rendered YAML file for each variant that will be built.
+    """
+    config = get_or_merge_config(None)
+    config.variant_config_files = variant_config_files
+    config.verbose = False
+    if not os.path.isfile(path):
```

Comment:
Should this also check that the file name is `meta.yaml`?

Reply:
That shouldn't be needed. We always pass the directory holding the meta.yaml file when rendering a recipe.